Mobile layout

afuhrtrumpet opened the pull request:
You can try it out at http://design-circle-test.meteor.com

afuhrtrumpet committed:
Added offscreen navbar for mobile

afuhrtrumpet committed:
Added full sidebar transition

afuhrtrumpet committed:
Merge remote-tracking branch 'origin/master' into mobile

afuhrtrumpet committed:
Merge branch 'bye-bye-scrollify' into mobile

afuhrtrumpet committed:
Improved hamburger icon

afuhrtrumpet committed:
Merged things

afuhrtrumpet committed:
Merge remote-tracking branch 'origin/bye-shitty-shit' into mobile

afuhrtrumpet committed:
Fixed navbar and modal

J3RN commented:
You are a good man. Let me know if you want a hand with this.
On Apr 7, 2015 13:07, "Alex Fuhr" notifications@github.com wrote:

> ## You can try it out at http://design-circle-test.meteor.com
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/CWDG/design-circle/pull/44
> Commit Summary
> - Added offscreen navbar for mobile
> - Added full sidebar transition
> - Merge remote-tracking branch 'origin/master' into mobile
> - Removed Scrollify and all associated scripts and styles
> - Merge branch 'bye-bye-scrollify' into mobile
> - Removed main.js - it wasn't needed
> - Improved hamburger icon
> - Added all anchors and links
> - Added first go at smooth scrolling to sections
> - I figured it out! :joy:
> - Merged things
> - Merge remote-tracking branch 'origin/bye-shitty-shit' into mobile
> - Fixed navbar and modal
> 
> File Changes
> - _M_ client/aboutpage.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-0 (2)
> - _M_ client/blog.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-1 (2)
> - _M_ client/calendar.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-2 (2)
> - _M_ client/contactform.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-3 (2)
> - _D_ client/css/jquery.fullPage.css
>   https://github.com/CWDG/design-circle/pull/44/files#diff-4 (186)
> - _M_ client/css/styles.css.scss
>   https://github.com/CWDG/design-circle/pull/44/files#diff-5 (126)
> - _M_ client/events.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-6 (2)
> - _M_ client/index.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-7 (18)
> - _D_ client/js/jquery.fullPage.min.js
>   https://github.com/CWDG/design-circle/pull/44/files#diff-8 (50)
> - _D_ client/js/jquery.scrollify.min.js
>   https://github.com/CWDG/design-circle/pull/44/files#diff-9 (1)
> - _D_ client/js/main.js
>   https://github.com/CWDG/design-circle/pull/44/files#diff-10 (7)
> - _M_ client/js/navbar.js
>   https://github.com/CWDG/design-circle/pull/44/files#diff-11 (25)
> - _A_ client/js/scroll.js
>   https://github.com/CWDG/design-circle/pull/44/files#diff-12 (12)
> - _M_ client/navbar.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-13 (10)
> - _M_ client/team.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-14 (2)
> - _M_ client/titlepage.html
>   https://github.com/CWDG/design-circle/pull/44/files#diff-15 (4)
> 
> Patch Links:
> - https://github.com/CWDG/design-circle/pull/44.patch
> - https://github.com/CWDG/design-circle/pull/44.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CWDG/design-circle/pull/44.

afuhrtrumpet commented:
The spacing still seems off on mobile, even after the 'bye-shitty-shit' merge. If you could fix that, things would be great (though maybe it's still supposed to be like that).

afuhrtrumpet committed:
Increased font sizes

afuhrtrumpet commented:
Text is bigger now. The about section extends past its bounds, but I think that's just a result of the full page CSS and it being too large.

afuhrtrumpet committed:
Fixed merge conflicts with horizontal

afuhrtrumpet commented:
I can't figure out what's causing the sections to be displayed on top of each other in the horizontal layout CSS. @J3RN, some help with this?

J3RN commented:
I’d really love to help, but I’m desperately trying to catch up on homework. If I find time, I will help out however I can.

> On Apr 20, 2015, at 10:49 PM, Alex Fuhr notifications@github.com wrote:
> 
> I can't figure out what's causing the sections to be displayed on top of each other in the horizontal layout CSS. @J3RN https://github.com/J3RN, some help with this?
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/CWDG/design-circle/pull/44#issuecomment-94621211.

afuhrtrumpet commented:
Alright. All I need to know is what you're doing to keep all the sections
at the top of the page.

On Mon, Apr 20, 2015, 11:34 PM Jonathan Arnett notifications@github.com
wrote:

> I’d really love to help, but I’m desperately trying to catch up on
> homework. If I find time, I will help out however I can.
> 
> > On Apr 20, 2015, at 10:49 PM, Alex Fuhr notifications@github.com
> > wrote:
> > 
> > I can't figure out what's causing the sections to be displayed on top of
> > each other in the horizontal layout CSS. @J3RN https://github.com/J3RN,
> > some help with this?
> > 
> > —
> > Reply to this email directly or view it on GitHub <
> > https://github.com/CWDG/design-circle/pull/44#issuecomment-94621211>.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CWDG/design-circle/pull/44#issuecomment-94628010.

J3RN commented:
So I just did a `float: left` on the `section` class, which all live in the precisely-sized-width `penultimate-container` class. That said, if it’s causing problems, the port to using `display: inline-block` is fairly trivial.

> On Apr 20, 2015, at 11:59 PM, Alex Fuhr notifications@github.com wrote:
> 
> Alright. All I need to know is what you're doing to keep all the sections
> at the top of the page.
> 
> On Mon, Apr 20, 2015, 11:34 PM Jonathan Arnett notifications@github.com
> wrote:
> 
> > I’d really love to help, but I’m desperately trying to catch up on
> > homework. If I find time, I will help out however I can.
> > 
> > > On Apr 20, 2015, at 10:49 PM, Alex Fuhr notifications@github.com
> > > wrote:
> > > 
> > > I can't figure out what's causing the sections to be displayed on top of
> > > each other in the horizontal layout CSS. @J3RN https://github.com/J3RN,
> > > some help with this?
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub <
> > > https://github.com/CWDG/design-circle/pull/44#issuecomment-94621211>.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/CWDG/design-circle/pull/44#issuecomment-94628010.
> > 
> > —
> > Reply to this email directly or view it on GitHub https://github.com/CWDG/design-circle/pull/44#issuecomment-94631728.

afuhrtrumpet committed:
Halp

afuhrtrumpet commented:
Okay, so I changed float to none, made the width of penultimate-container 100vw, and changed the display to 'inline-block'. I don't see any absolute or fixed positioning, so I have no idea what's doing this.

J3RN commented:
Are you wanting the `section`s to stack? The `penultimate-container` was `600vw` to allow 6 sections of `100vw` to fit inside.

As a side note, `inline-block` puts a small space in between each element. This is literally a space, as in typography, and it's size is dependent on the font size. To made the space between elements 0, you can set `font-size: 0;` for `penultimate-container` and then reset the font size to whatever it is currently inside `.main-nav` and `.section`.

afuhrtrumpet commented:
All I want is for the sections to be displayed vertically and not on top of each other (how the old layout was).

J3RN commented:
If you just want them to stack, there's no need for `display: inline-block`, you can just do `display: block`, which is actually the default

afuhrtrumpet commented:
That's still not changing the fact that they don't stack. What CSS do you have that is aligning each `.section` at the top of the page?

J3RN commented:
There is none. They are placed that way because they float left, and fit in
the penultimate container - the same way grid layouts get columns to be
placed side-by-side.
On Apr 21, 2015 12:56 PM, "Alex Fuhr" notifications@github.com wrote:

> That's still not changing the fact that they don't stack. What CSS do you
> have that is aligning each .section at the top of the page?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CWDG/design-circle/pull/44#issuecomment-94869560.

afuhrtrumpet committed:
Got a little better

afuhrtrumpet commented:
I was able to get something going by setting `height=auto` for `#content`, but still the about page is showing about 50% down the page for some reason and scrolling is broken.

afuhrtrumpet commented:
I also can't scroll and I'm not sure why, as `div#content` clearly has `overflow-y: scroll` on it...

afuhrtrumpet committed:
Can't scroll?
